spec/services/nebraska/daily/revenue_calculator_spec.rb

```diff
@@ -107,6 +107,132 @@
   let!(:child_fih_child_approval) { child_fih.child_approvals.first }
 
   describe '#call' do
+    context 'when called after qris status update (7-1-2022)' do
+      it 'matches hourly rating based on qris_rate if there is one and does not apply qris bump' do
+        hourly_ldds_rate.update!(qris_rating: 'step_three')
+        business_ldds.update!(qris_rating: 'step_three')
+        child_ldds.reload
+        child_ldds_child_approval.reload
+        expect(
+          described_class.new(
+            child_approval: child_ldds_child_approval,
+            date: '2022-7-1'.to_date,
```

Comment:
So this is great, declarative date expectation.  If you look at the rates and effective_dates above, they're all listed as (for example) `1.month.ago` or `3.months.ago` - several of these are likely going to fail once we are 1 month or 3 months away from July 1, 2022 when the change went into effect.  You'll want to go through all of these examples and make sure that if there are expectations including qris bumps and ratings that the dates are set appropriately to calculate qris bumps before 7/1/2022, or calculate without the exponential bump after.  This is going to be a big change to this spec, make sure you're checking each example.  One of the things I do sometimes if I think specs are going to fail based on time, is literally change the time on my computer's clock (set it to 6 months in the future or 6 months in the past) and then run the specs again.  You can also do it with `travel_to` as we've seen in other specs.  This is one of the trickiest parts of handling specs in this app, all the date math - please let me know if you need help!